packages/core/src/components/Switch.tsx

```diff
@@ -116,13 +117,19 @@ function Row({
   theme,
   ...rest
 }: Props & RowProps) {
-  const [checked, setChecked] = React.useState(value);
+  const [checked, setChecked] = React.useState(value || defaultValue);
```

Comment:
```suggestion
  const [checked, setChecked] = React.useState(value != null ? value : defaultValue);
```

Just thinking that since value is a bool, as written this would ignore `false`

Reply:
Ooh, good point!